### PR TITLE
FIX: Update for new pep8

### PR DESCRIPTION
**Pull request by Eric89GXL**
@almarklein ready for review/merge assuming Travis is happy. The PEP8 program was updated with new rules, and this ignores one of them and brings us into compliance with the others.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
@almarklein good to go, feel free to merge and re-trigger your Travis run

File: vispy/gloo/program.py [hunk 1]
```diff
@@ -301,7 +301,10 @@ def __setitem__(self, name, data):
             elif kind == 'attribute':
                 # Is this a constant value per vertex
                 is_constant = False
-                isscalar = lambda x: isinstance(x, (float, int))
+
+                def isscalar(x):
+                    return isinstance(x, (float, int))
+
```
Comment (almarklein) [inline vispy/gloo/program.py]: why the F are lambdas bad according to flake? I personally find the lambda more pretty (and shorter!) in this case.

Perhaps also don't apply this rule?

Comment (almarklein) [inline vispy/gloo/program.py]: Are lambda's always considered bad? I cannot imagine this was the only lambda in our code base ...

**Comment by almarklein** [comment]
Thanks for the quick fix by the way!

**Comment by rossant** [comment]
I think `a = lambda...` is bad but not `f(lambda...)`

File: vispy/gloo/program.py (hunk 1 above)
Comment (Eric89GXL) [inline vispy/gloo/program.py]: No, they aren't always bad, they are just not good when you are trying to assign a function to a variable name directly -- that is better done through `def`. See:

http://stackoverflow.com/questions/25010167/e731-do-not-assign-a-lambda-expression-use-a-def

Comment (almarklein) [inline vispy/gloo/program.py]: Still find lambda prettier in this case, but I'll merge for now :)

**Pull request merged**